client/my-sites/media-library/test/content.jsx

```diff
@@ -85,29 +84,29 @@ describe( 'MediaLibraryContent', () => {
 				mediaValidationErrorTypes: [],
 				source: 'google_photos',
 			};
-			const wrapper = getMediaContentInstance();
+			const content = new MediaLibraryContent();
```

Comment:
As discussed during pairing the other day, this is an acceptable compromise when we really want to test the methods, given the complexity necessary to setup all needed data to get to that scenario.